projects/js-packages/charts/src/components/pie-semi-circle-chart/pie-semi-circle-chart.tsx

```diff
@@ -11,32 +11,43 @@ import { withResponsive } from '../shared/with-responsive';
 import { BaseTooltip } from '../tooltip';
 import styles from './pie-semi-circle-chart.module.scss';
 import type { BaseChartProps, DataPointPercentage } from '../../types';
-
 interface PieSemiCircleChartProps extends BaseChartProps< DataPointPercentage[] > {
+	/**
+	 * Size of the chart in pixels
+	 */
+	size?: number;
+
+	/**
+	 * Thickness of the pie chart. A value between 0 and 1
+	 */
+	thickness?: number;
+
 	/**
 	 * Label text to display above the chart
 	 */
-	label: string;
+	label?: string;
 	/**
 	 * Note text to display below the label
 	 */
-	note: string;
+	note?: string;
 	/**
 	 * Direction of chart rendering
 	 * true for clockwise, false for counter-clockwise
 	 */
 	clockwise?: boolean;
+
 	/**
-	 * Thickness of the pie chart. A value between 0 and 1
+	 * Deprecated
 	 */
-	thickness?: number;
+	width?: number;
 }
 
 type ArcData = PieArcDatum< DataPointPercentage >;
 
 const PieSemiCircleChart: FC< PieSemiCircleChartProps > = ( {
 	data,
-	width = 500, //TODO: replace when making the components responsive
+	size = 500,
+	width = 500, // Deprecated. TODO: Remove when Woo Anaytics usage is updated.
```

Comment:
Woo Analytics is already consuming the package, so we should always keep the compatibility if possible.

That said the property doesn't seem to be used yet. Let me revert my changes.